CMakeLists.txt

```diff
@@ -2,7 +2,15 @@ cmake_minimum_required(VERSION 2.8)
 
 set(PROJECT_NAME mc-mini)
 set(PROJECT_VERSION 1.1.0)
-project(${PROJECT_NAME})
+
+if(CMAKE_VERSION LESS 3.0)
```

Comment:
That would do it! Looks good. Just clean up the trailing whitespace at least and I can merge in this request.